kademlia/network.py

```diff
@@ -216,9 +219,12 @@ def saveStateRegularly(self, fname, frequency=600):
 
         Args:
             fname: File name to save retularly to
-            frequencey: Frequency in seconds that the state should be saved.
+            frequency: Frequency in seconds that the state should be saved.
                         By default, 10 minutes.
         """
-        loop = LoopingCall(self.saveState, fname)
-        loop.start(frequency)
-        return loop
+        asyncio.ensure_future(self._saveStateRegularly(fname))
```

Comment:
Not sure why this line is here - what are you looking to do here?

Reply:
Could you just make this a call to `self.saveState(fname)`?

Reply:
You are totally right! I'm sorry, I've never used asyncio before 🙃 .

Reply:
No worries!